core/interfaces.go

```diff
@@ -149,3 +149,11 @@ type DNSResolver interface {
 	LookupCAA(string) ([]*dns.CAA, time.Duration, error)
 	LookupMX(string) ([]string, time.Duration, error)
 }
+
+// PolicyAuthorityDatabase is the public interface to the white/blacklists
+type PolicyAuthorityDatabase interface {
```

Comment:
Looking at this again, it seems that this interface is only used internally to PolicyAuthority, so should be defined there.